components/dialog/dialog-mixin.js

```diff
@@ -281,6 +281,9 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 		// native dialog backdrop does not prevent body scrolling
 		this._bodyScrollKey = preventBodyScroll();
 
+		if (document.activeElement) {
```

Comment:
Does this happen with both the native dialog and custom?  Do you know what is placing the focus on the button... we should not be trying to place focus until below... but it will be one frame from this code.

Reply:
At the top of the demo page there is a commented line of code if you wish to test with the custom dialog impl in Chrome.

Reply:
Its caused by the native dialog. When running the custom dialog its fine.

Reply:
The other option I considered here was increasing the `tabindex` of the close button so that its considered last which also fixes this issue with native dialogs.

Reply:
I'd lean away from the `tabindex` approach since it could interfere with the cycling of the focus within the dialog.  I think the `blur` is fine, but perhaps add a comment similar to the one above... `\\ native dialog... tooltip... ` since it won't be otherwise clear why this is needed.